[{"ContainerToolsException.Message":".FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Projects\\\\workshop-docker\\\\compose\\\\obj\\\\Docker\\\\docker-compose.vs.release.partial.g.yml'","ContainerToolsException.ErrorCode":"DT1001","ContainerToolsException.StackTrace":"   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerComposeClient.<ExecuteAsync>d__9.MoveNext()\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerComposeClient.<DownAsync>d__6.MoveNext()\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerWorkspace.<ComposeDownContainersAsync>d__26.MoveNext()\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.BuildTasks.DockerBuildTask.<ExecuteAsync>d__28.MoveNext()","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.CommandLine.Program":"docker-compose","ContainerToolsException.CommandLine.TelemetrySafeArguments":"","ContainerToolsException.CommandLine.ExitCode":"1","ContainerToolsException.CommandLine.ErrorOutput":".FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Projects\\\\workshop-docker\\\\compose\\\\obj\\\\Docker\\\\docker-compose.vs.release.partial.g.yml'"}]